src/index.html

```diff
@@ -12,7 +12,67 @@
       href="./style.css"
     />
   </head>
+
   <body>
-    <h1>Stars</h1>
+    <ul class="stars stars--0">
+
+      <li class="stars__star"></li>
+      <li class="stars__star"></li>
+      <li class="stars__star"></li>
+      <li class="stars__star"></li>
+      <li class="stars__star"></li>
+
+    </ul>
+
+    <ul class="stars stars--1">
+
+      <li class="stars__star"></li>
+      <li class="stars__star"></li>
+      <li class="stars__star"></li>
+      <li class="stars__star"></li>
+      <li class="stars__star"></li>
+
+    </ul>
+
+    <ul class="stars stars--2">
+
+      <li class="stars__star"></li>
+      <li class="stars__star"></li>
+      <li class="stars__star"></li>
+      <li class="stars__star"></li>
+      <li class="stars__star"></li>
+
+    </ul>
+
+    <ul class="stars stars--3">
+
+      <li class="stars__star"></li>
+      <li class="stars__star"></li>
+      <li class="stars__star"></li>
+      <li class="stars__star"></li>
+      <li class="stars__star"></li>
+
+    </ul>
+
+    <ul class="stars stars--4">
+
+      <li class="stars__star"></li>
+      <li class="stars__star"></li>
+      <li class="stars__star"></li>
+      <li class="stars__star"></li>
+      <li class="stars__star"></li>
+
+    </ul>
+
+    <ul class="stars stars--5">
+
+      <li class="stars__star"></li>
+      <li class="stars__star"></li>
+      <li class="stars__star"></li>
+      <li class="stars__star"></li>
+      <li class="stars__star"></li>
```

Comment:
Add empty lines between multiline sibling blocks of HTML. There should be an empty line between each <ul> block to improve readability.